src/components/SelectFileType/SelectFileType.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+
+    <div 
+        @click="selectFile()"
+        class="inline-flex items-center border border-steel text-center h-24 w-24 rounded-md border-2 relative cursor-pointer overflow-hidden justify-center"
+        :class="props.selected ? 'bg-indigo fill-white' : ' bg-white fill-gray-500'">
```

Comment:
You don't need to use `props` in the template, only the script. It works, but not necessary. Do this for other instances below

```suggestion
        :class="selected ? 'bg-indigo fill-white' : ' bg-white fill-gray-500'">
```